data_model/src/trigger.rs

```diff
@@ -41,7 +41,6 @@ pub mod model {
         IntoSchema,
     )]
     #[getset(get = "pub")]
-    #[ffi_type]
```

Comment:
I don't get this removal of `ffi_type`. Can't user use `TriggerId` in their smart contract?

Reply:
I was just removing `#[model]` macro from the places which broke my compilation. Although it can be restored back. 

Do you think we need model and ffi in data_model?

Reply:
if a user can access these structs through both smart contract and client, then yes we need this back

Reply:
Done